storage/innobase/xtrabackup/doc/source/prerequisites/configuring.rst

```diff
@@ -0,0 +1,44 @@
+.. _configuring:
+
+Configuring XtraBackup
+======================
+
+Setting options configures the *Percona XtraBackup*. These options behave
+exactly like the standard |MySQL| program options: they can be specified either at
+the command-line or through a configuration file like `/etc/my.cnf`.
+
+Please see the :ref:`option and variable reference
```

Comment:
Please is not recommended for use: https://developers.google.com/style/tone?hl=en#politeness-and-use-of-please

Reply:
Agreed. Please removed.